### ISSUE TITLE
My avatar is missing

### ISSUE THREAD

**Issue by tjreo** [jeena/Bungloo#45]
Everyone who is posting today is showing the proper avatar in Tentia. Except for me. 

My avatar is the default blank and white tent. My entity "^tjreo.tent.is" is not getting changed into "Tony Jensen" in posts either. 

Is it possible that MY profile information somehow didn't get updated? I went to preferences thinking I'd be able to see who I am along with my profile info but I don't see it there.

**Comment by jeena** [comment]
Something like that should show up in the Console.app, could you have a look at it there? And can you reach your profile JSON from your own computer? I myself am seeing your avatar and name.

**Comment by tjreo** [comment]
Here is my profile JSON as downloaded from my computer. (Hey, check it out. We have cursors coming. I saw that in the blog post earlier but hadn't seen this yet.)

``` json
{
  "https://tent.io/types/info/tent-status/v0.1.0": {
    "background_image_url": "http://subtlepatterns.subtlepatterns.netdna-cdn.com/patterns/otis_redding.png",
    "cursors": {
      "mentions": {
        "post_entity": "https://daniel.tent.is",
        "post_id": "shtgsz"
      }
    },
    "permissions": {
      "public": true
    }
  },
  "https://tent.io/types/info/basic/v0.1.0": {
    "name": "Tony Jensen",
    "avatar_url": "http://www.stellarwell.com/wp3/wp-content/uploads/2012/09/tony_headshot_sq_gs_100px.jpg",
    "birthdate": "",
    "location": "Denver, Colorado",
    "gender": "male",
    "bio": "Husband, Father, Enterprise software developer, Space explorer, Producer of sarcasm, Denver is my playground.",
    "url": "stellarwell.com",
    "permissions": {
      "public": true
    }
  },
  "https://tent.io/types/info/core/v0.1.0": {
    "entity": "https://tjreo.tent.is",
    "licenses": [

    ],
    "servers": [
      "https://tjreo.tent.is/tent"
    ],
    "permissions": {
      "public": true
    }
  }
}
```

**Comment by jeena** [comment]
Yes I saw the same behaviour happening on my brothers computer yesterday that some profiles just would not load sometimes. I will have to look into that and perhaps make some retry to fetch it.

I saw actually two cursors in mine, one for mentions and one for the timeline posts, this needs to be implemented as soon as possible too, but I think it will too be implemented after a Linux version is out there.
